### ISSUE TITLE
Update notifications_pref table

### ISSUE THREAD

**Issue by Spuds** [elkarte/Elkarte#3530]
This is a todo for the updater (when there is one)

The notifications_pref table changed column ```notification_level```  to ```notification_type``` moving from int to  text.  There also needs to be a translation from the old numeric to string.

**Comment by emanuele45** [comment]
oh... ehm... :eyes:

**Comment by Spuds** [comment]
After the column is altered (name and value type) the translation appears to be
0  => ["none"]
1 => ["notification"]
2  => ["email","notification"]
3 => ["emaildaily","notification"]
4  => ["emailweekly","notification"]
```
UPDATE elkarte_notifications_pref
SET notification_type = '["none"]'
WHERE notification_type = '0'
```

**Comment by emanuele45** [comment]
I was just making fun of myself for having forgotten to create the 
upgrade and the conversions. :D

**Comment by emanuele45** [comment]
I forgot to mention it in the commit, but this one should have been fixed by c83ff1e5158e1c1de7115c0e6b6f798eaff5a2e2 I hope.

**Comment by Spuds** [comment]
Awesome ... I noticed you were doing some changes to the upgrade scripts 😄
